kubernetes/redpanda/templates/service.yaml

```diff
@@ -0,0 +1,20 @@
+{{- $envAll := . }}
+{{- range untilStep 0 (.Values.replicaCount|int) 1 }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "redpanda.fullname" $envAll }}-{{ . }}
+spec:
+  ports:
+    - name: rpc
+      protocol: TCP
+      port: {{ $envAll.Values.redpanda.advertised_rpc_api.port }}
+      targetPort: {{ $envAll.Values.redpanda.advertised_rpc_api.port }}
+    - name: kafka
```

Comment:
We should probably consider port naming (also for `rpc`): https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/

No reason to not be compatible. Either append `-tcp` or allow customisation.

Reply:
Fixed in: https://github.com/oranda/deployment-automation/pull/3